### ISSUE TITLE
tv4play not working

### ISSUE THREAD

**Issue by jula420** [spaam/svtplay-dl#1557]
svtplay-dl versions: 4.25

Ubuntu 22.04 python version 3.10.12

tv4play seems to have lost the "allSeasonLinks"

**Comment by spaam** [comment]
you forgot to share a link with this issue.  i cant test and see if i see the same issue.

**Comment by jula420** [comment]
svtplay-dl -u "xxx" -p "yyy" https://www.tv4play.se/program/6cd3abf7ba31c6d07f4e/f%C3%A5ngarna-p%C3%A5-fortet

Or any other tv4 link

**Comment by jula420** [comment]
With the -A flag that I forgot above

**Comment by spaam** [comment]
it was an easy fix
```
imagine svtplay-dl-dev/lib ‹master*› » python3 -m svtplay_dl https://www.tv4play.se/program/6cd3abf7ba31c6d07f4e/f%C3%A5ngarna-p%C3%A5-fortet -A                         130 ↵
INFO: Episode 1 of 16
INFO: Url: https://www.tv4play.se/video/163442828f54d5045d2d
```

**Comment by Gurkan911** [comment]
I'm probably missing something but I thought that maybe you would make the fix available via the download page: https://svtplay-dl.se/download/snapshots/ but on that page, there is nothing made available since 18-jul-2023. Would it be possible with an update, please?

**Comment by jula420** [comment]
tv4play has now changed their login so not working at all.

**Comment by spaam** [comment]
original issue was fixed in 4.26

@jula420 , you dont share any info on what version you are using so i assume you are using an old version. open a new issue if its 4.26.
